src/Translation/locales/ja.json

```diff
@@ -149,7 +149,7 @@
     "en": "English",
```

Comment:
Can we translate Spanish, French... and the rest of the languages to Japanese too?

Reply:
I just updated them:) I was wondering if they're written in Japanese, a non-Japanese reader can't read and go back to their language?